### PR TITLE
Added template for ad_master_1replica_1client

### PR DESCRIPTION
**Pull request by amore17**

### CODE REVIEW

**Comment by amore17** [comment]
Required for https://github.com/freeipa/freeipa/pull/7400/

**Review by flo-renaud — COMMENTED** [review]
Hi @amore17 
please find one inline comment.

File: templates/vagrantfiles/Vagrantfile.ad_master_1repl_1client [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    end
+
+    config.vm.define "replica0"  do |replica0|
+    end
+
+    config.vm.define "client0"  do |client0|
+        client0.vm.provider "libvirt" do |domain,override|
+            domain.memory = 1250
+        end
+    end
```
Comment (flo-renaud) [inline templates/vagrantfiles/Vagrantfile.ad_master_1repl_1client:84]: There is a final "end" missing (corresponding to the section starting line 11 `Vagrant.configure(2) do |config|`)

**Review by flo-renaud — APPROVED** [review]
LGTM

**Pull request merged**